java/core/src/main/java/com/microsoft/bond/NullableBondType.java

```diff
@@ -82,6 +82,11 @@ protected final TValue newDefaultValue() {
     }
 
     @Override
+    protected final TValue cloneValue(TValue value) {
+        return value == null ? null : this.valueType.cloneValue(value);
```

Comment:
If someone ends up cloning a struct with a non-nullable field which is set to null, how will that exception/stacktrace look?